providers/dns/cloudns/internal/client.go

```diff
@@ -130,7 +135,7 @@ func (c *Client) FindTxtRecord(zoneName, fqdn string) (*TXTRecord, error) {
 		}
 	}
 
-	return nil, fmt.Errorf("ClouDNS: no existing record found for %q", fqdn)
+	return nil, nil
```

Comment:
Why aren't you returning an error here anymore?

Reply:
I manage the `nil` inside the provider, seems like a better way to manage this part.

Reply:
the method `FindTxtRecord` is only use for the clean up.